test/CMakeLists.txt

```diff
@@ -48,14 +48,55 @@ add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/small
 add_custom_target(symlink_libsmall_test_binaries ALL
     DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/small)
 
+add_custom_target(unit-test
+    DEPENDS unit-test-bin
```

Comment:
Is there any difference from `DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/unit`? I'm able to run this target from console this way: `make -C test/unit`.

The same about other custom targets.

Reply:
:(